jenkins/custom-ci/generic/cortx-rgw-integration.groovy

```diff
@@ -11,6 +11,12 @@ pipeline {
         string(name: 'CORTX_RGW_INTEGRATION_BRANCH', defaultValue: 'main', description: 'Branch for cortx-rgw-integration.')
         string(name: 'CUSTOM_CI_BUILD_ID', defaultValue: '0', description: 'Custom CI Build Number')
         // Add os_version parameter in jenkins configuration
+
+        choice(
+            name: 'ENABLE_ADDB_PLUGIN',
+                choices: ['yes', 'no'],
```

Comment:
Please put default choice as "no". Developers can select when they need to build addb plugin.

Reply:
Done

Reply:
@Abhirup-A-G, get it clarified with developers if they want default as `yes` or `no` each time.

Reply:
They wanted it as No